### PR TITLE
feat: update recaptcha demosite front end

### PR DESCRIPTION
**Pull request by morewry**
## Description

Update to latest frontend code. The change fixes 2 bugs identified during bughunt.

1. Fixes the tab navigation to blank page (b/278565964)
2. Button says "Play the game" after the results are obtained in the comment page. (b/278563766)

https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3159
https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9746
https://github.com/GoogleCloudPlatform/java-docs-samples/pull/7921

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: recaptcha_enterprise/demosite/app/static/demo-6df0841a.js [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
+purpose with or without fee is hereby granted.
+
+THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+PERFORMANCE OF THIS SOFTWARE.
+***************************************************************************** */
```
Comment (dandhlee) [inline recaptcha_enterprise/demosite/app/static/demo-6df0841a.js:14]: Any reason why this is replaced with the Microsoft license? I want to err on the side of caution even though I assume this is a test file - we will need legal approval for updates like these unless you're appending the Microsoft license as needed.

File: recaptcha_enterprise/demosite/app/static/demo-6df0841a.js [hunk 2]
```diff
@@ -3723,10 +3743,19 @@ class RecaptchaDemo extends s {
     this._score = value;
     this.requestUpdate("score", oldValue);
     const buttonElement = document.querySelector("recaptcha-demo > button");
-    if (buttonElement && this._score && this.step !== "comment") {
-      window.setTimeout(() => {
-        buttonElement.innerText = "Go to next demo";
-      }, 100);
+    if (buttonElement && this._score) {
+      // TODO: redesign per https://b.corp.google.com/issues/278563766
```
Comment (dandhlee) [inline recaptcha_enterprise/demosite/app/static/demo-6df0841a.js:3747]: Use b/278563766 - although Buganizer is public, the link shown here is an internal link versus public-facing link (https://issuetracker.google.com/issues)
```suggestion
      // TODO: redesign per b/278563766
```

File: recaptcha_enterprise/demosite/app/static/demo-6df0841a.js [hunk 3]
```diff
@@ -4113,7 +4143,7 @@ class RecaptchaDemo extends s {
             <h4 class="h1">Pattern</h4>
             <h5 class="h2">Prevent malicious log in</h5>
             <p>
-              Add reCAPTCHA to user actions like logging in to prevent malicious
+              Add reCAPTCHA to user actions like logging in to prevent malcious
```
Comment (dandhlee) [inline recaptcha_enterprise/demosite/app/static/demo-6df0841a.js:4146]: Is the typo intentional?

**Review by dandhlee — CHANGES_REQUESTED** [review]

File: recaptcha_enterprise/demosite/app/static/demo-6df0841a.js (hunk 1 above)
Comment (Sita04) [reply recaptcha_enterprise/demosite/app/static/demo-6df0841a.js:14]: No, this is a built file and I didn't get the chance earlier to make a commit and update the license.

File: recaptcha_enterprise/demosite/app/static/demo-6df0841a.js (hunk 2 above)
Comment (Sita04) [reply recaptcha_enterprise/demosite/app/static/demo-6df0841a.js]: Done.. thanks!

File: recaptcha_enterprise/demosite/app/static/demo-6df0841a.js (hunk 3 above)
Comment (Sita04) [reply recaptcha_enterprise/demosite/app/static/demo-6df0841a.js]: Nope, thanks for the catch.

**Review by Sita04 — APPROVED** [review]

**Review by dandhlee — APPROVED** [review]
Thank you!

**Review by HansEngebretsen — APPROVED** [review]
LGTM

**Pull request merged**